events/syscall/program_run_with_disallowed_http_proxy_env.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+	ProgramRunWithDisallowedHttpProxyEnv,
+	events.WithDisabled(), // this rules is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func ProgramRunWithDisallowedHttpProxyEnv(h events.Helper) error {
+	cmd := exec.Command("curl", "http://example.com")
```

Comment:
Can we always do a `exec.LookPath` and then give an error if `curl` is not present? Like eg: https://github.com/falcosecurity/event-generator/blob/4c28e177dc3df2bce8124f5c8ac7cd4079b5cde7/events/syscall/launch_suspicious_network_tool_on_host.go#L26

Reply:
Yepp got it! Just did the changes